tests/dummy/app/templates/snippets/buttons-dropdown.hbs

```diff
@@ -0,0 +1,9 @@
+{{md-btn-dropdown text='Button' activates="dropdown-target"}}
+
+<!-- Dropdown Structure -->
+<ul id='dropdown-target' class='dropdown-content'>
```

Comment:
Uh... You are not allowed  to give elements the same id's, unlike classes.

So multiple drop downs may target the same Id. I think we leave it to the dev to make sure thee targets have unique id's?

This is inline with how the material js handles it (it does not).

Am I missing something?

However if the target element is in dom when the material plugin is inited might be a problem.



> Op 3 jul. 2015 om 18:32 heeft Mike North <notifications@github.com> het volgende geschreven:
> 
> In tests/dummy/app/templates/snippets/buttons-dropdown.hbs:
> 
> > @@ -0,0 +1,9 @@
> > +{{md-btn-dropdown text='Button' activates="dropdown-target"}}
> > +
> > +<!-- Dropdown Structure -->
> > +<ul id='dropdown-target' class='dropdown-content'>
> Imagine that you have several of these button dropdowns on the page at once. You will have several ul#dropdown-target elements, and there's really no telling which buttons will target which (or all?).
> 
> You need to generate an ID off of something unique to the instance of the component. I like to use the component's root element ID, since that's guaranteed to be unique. See https://github.com/truenorth/ember-cli-materialize/blob/master/addon/components/md-navbar.js#L24-L28
> 
> —
> Reply to this email directly or view it on GitHub.
>